packages/components/inputs/input-utils/src/accessible-hidden-input.styles.ts

```diff
@@ -1,13 +1,9 @@
 import { css } from '@emotion/react';
 
 const accessibleHiddenInputStyles = css`
-  pointer-events: none;
-  height: 100%;
-  left: 0;
-  opacity: 0.0001;
-  position: absolute;
+  left: -9999px;
+  position: fixed;
   top: 0;
-  width: 100%;
```

Comment:
Ah, forgot to mention this. It's not related to this PR, but when we were looking into why tests were failing, these stylings were looking weird. Then the OCD kicked in. 😄 
Is it better if I removed this since it's not related?